src/ServiceInsight.Tests/SearchViewModelTests.cs

```diff
@@ -11,20 +12,24 @@
     using Shouldly;
 
     [TestFixture]
-    public class SearchViewModelTests
+    public sealed class SearchViewModelTests : IDisposable
```

Comment:
It's because they implement `IDisposable`. If we don't seal them then we have to implement the [entire pattern](https://msdn.microsoft.com/en-us/library/fs2xkftw.aspx) or otherwise yet another warning is given.

Generally, I agree regarding `sealed` classes and I'd be hesitant to seal anything in the production code, but these are just tests. If we ever need to inherit from them (highly unlikely) then we can unseal them and pay any further `IDisposable` tax at that time. We could also introduce Fody.Janitor instead but I think it's overkill for addressing these two violations.